.github/ci/packages.apt

```diff
@@ -2,18 +2,18 @@ binutils-dev
 libgflags-dev
 libignition-cmake2-dev
 libignition-common4-dev
-libignition-fuel-tools7-dev
+libignition-fuel-tools8-dev
```

Comment:
I wonder why this is here

Reply:
it's an indirect dependency. it probably doesn't need to be there since ign-launch doesn't `find_package` it